amivapi/events/endpoints.py

```diff
@@ -8,184 +8,253 @@
 As soon as we switch to mongo this will only have the schema.
 """
 
-from sqlalchemy import (
-    Column,
-    ForeignKey,
-    DateTime,
-    Unicode,
-    UnicodeText,
-    Text,
-    Boolean,
-    Integer,
-    CHAR)
-from sqlalchemy.ext.hybrid import hybrid_property
-from sqlalchemy.orm import relationship
-
-from amivapi import utils
-
-
-class Event(utils.Base):
-    """Event model."""
-
-    __description__ = {
-        'general': "An Event is basically everything happening in the AMIV. "
-        "All time fields have the format YYYY-MM-DDThh:mmZ, e.g. "
-        "2014-12-20T11:50:06Z",
-        'methods': {
-            'GET': "You are always allowed, even without session, to view "
-            "AMIV-Events"
+from amivapi.settings import EMAIL_REGEX
+from .authorization import EventSignupAuth
+
+
+eventdomain = {
+    'events': {
+        'description': {
+            'fields': {
+                'additional_fields':
+                'must be provided in form of a JSON-Schema. You can add here '
+                'fields you want to know from people signing up going further '
+                'than their email-address',
+                'allow_email_signup': 'If False, only AMIV-Members can sign '
+                'up for this event',
+                'price': 'Price of the event as Integer in Rappen.',
+                'spots': "For no limit, set to '0'. If no signup required, "
+                "set to '-1'. Otherwise just provide an integer."
+            },
+            'general': 'An Event is basically everything happening in the '
+            'AMIV. All time fields have the format YYYY-MM-DDThh:mmZ, e.g. '
+            '2014-12-20T11:50:06Z',
```

Comment:
didn't this change for JS reasons? Anyway, the example and the description are not consistent